### DISCUSSION TITLE
gh-pages wont show my site  allthough all checks completed

### DISCUSSION THREAD

**Discussion by nrbnlulu** [squidfunk/mkdocs-material#3955] [Q&A] [answered]
repo is https://github.com/nrbnlulu/strawberry-django-auth
pages action https://github.com/nrbnlulu/strawberry-django-auth/blob/main/.github/workflows/gh-pages.yml
make command https://github.com/nrbnlulu/strawberry-django-auth/blob/1d3deca496910cb0b2938d188a789d9953019fbf/Makefile#L41
site link https://nrbnlulu.github.io/strawberry-django-auth/
for hours github shows this:
![image](https://user-images.githubusercontent.com/88795475/170511546-c202d3e4-1e16-4369-9f4d-793987471b98.png)
like my site is being proceeded forever

**Comment by lin-ycv** [comment]
You're deploying a branch that doesn't exist?
![image](https://user-images.githubusercontent.com/60565737/170524979-ace71984-5bce-4723-b040-1a2c27829ac7.png)

**Reply by facelessuser** [reply]
@lin-ycv Did you create a `gh-pages` branch? I'd create an orphaned branch named `gh-pages`. Go into your GitHub settings for that repo and make sure it is configured to build sites from the `gh-pages` branch.

**Reply by nrbnlulu** [reply]
I just deleted that brach because before i was trying to import the site to readthedocs..
I noticed somthing strange though github redirects me to a url that dosnt exists

**Comment by nrbnlulu** [comment]
Dang! Its working now! Probabbly readthedocs messed up this branch..

**Comment by nrbnlulu** [comment, marked as answer]
turns out that before i have changed the default url for the page and restored it afterwards and firfox cached the redirection from github therefore redirecting me to a page that dosnt exists...  

solution was to clear github cache from firefox
if you encounter such problme try opning the page link in private mode.
